JASP-Desktop/po/jasp_nl.po

```diff
@@ -15498,6 +15470,175 @@ msgctxt "AnalysisFormExpander|"
 msgid "Remove this analysis"
 msgstr "Verwijder deze analyse"
 
+msgctxt "PrefsUI|"
+msgid "Disable Animations"
+msgstr "Schakel Animaties uit"
+
+msgctxt "PrefsUI|"
+msgid ""
+"Turns off all animations, this is implied when \"Safe Graphics Mode\" is on."
+msgstr ""
+"Schakelt alle animaties uit, dit is impliciet wanneer \"Veilige Grafische "
+"Mode\" aan staat."
+
+msgctxt "PrefsUI|"
+msgid "Already disabled animations because \"Safe Graphics Mode\" is on"
+msgstr "Animaties al uitgezet omdat de \"Veilige Grafische Mode\" aan staat"
+
+msgctxt "EngineRepresentation|"
+msgid "The engine crashed while trying to run the filter..."
+msgstr "De engine is gecrashed tijdens het pogen een filter te draaien..."
```

Comment:
tijdens een poging een filter te draaien...

Reply:
Of gewoon "De engine is gecrashed tijdens het uitvoeren van de filter" want we hebben zoiezo maar 1 filter.

Reply:
En de andere ëngine gecrashed" vertaling laten het pogen ook achterwege.

Reply:
OK